libs/domains/services/data-access/src/lib/domains-services-data-access.ts

```diff
@@ -27,15 +31,18 @@ const applicationsApi = new ApplicationsApi()
 const containersApi = new ContainersApi()
 const databasesApi = new DatabasesApi()
 const jobsApi = new JobsApi()
+const helmsApi = new HelmsApi()
 
 const applicationMainCallsApi = new ApplicationMainCallsApi()
 const containerMainCallsApi = new ContainerMainCallsApi()
 const databaseMainCallsApi = new DatabaseMainCallsApi()
 const environmentMainCallsApi = new EnvironmentMainCallsApi()
 const jobMainCallsApi = new JobMainCallsApi()
+const helmCallsApi = new HelmMainCallsApi()
 
 const applicationDeploymentApi = new ApplicationDeploymentRestrictionApi()
 const jobDeploymentApi = new JobDeploymentRestrictionApi()
+const helmDeploymentApi = new HelmDeploymentRestrictionApi()
 
 // Use this type in param instead of ServiceTypeEnum
```

Comment:
Not true, because you use this kind `serviceType: ServiceTypeEnum.HELM as const,` for example, on your data-access, no ?

Reply:
For `serviceType: ServiceTypeEnum.HELM as const` the `as const` act like a string cast and avoid the introduction of another type union / enum. It's mainly for compatibility reason to use redux and react-query fetched services in data-access. It should be removed when we will be 100% relying on react-query.

Reply:
ok maybe it should be clarified in the comment 👍